CHANGELOG.md

```diff
@@ -4,9 +4,11 @@ All notable changes to this project will be documented in this file.
 
 ## Unreleased
 
+##  - 2024-04-26
```

Comment:
No need for a date here. The date is set by the merging person on the day the merge takes place.